app/models/list.rb

```diff
@@ -0,0 +1,33 @@
+# == Schema Information
+#
+# Table name: lists
+#
+#  id         :integer          not null, primary key
+#  name       :string           not null
+#  user_id    :integer          not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class List < ApplicationRecord
+  validates :name, :user_id, presence: true
+  validates :user_id, uniqueness: { scope: :name, message: "list titles must be unique"}
```

Comment:
@mcgannco This is an interesting uniqueness check. What was your reasoning when adding this validation?

Reply:
Hi Corey,
The purpose of this uniqueness check was to have unique list titles for every user.  For example, if I was user A and I had a list titled "errands", I could not create another list called "errands".  This was a feature I thought could be useful since it would get confusing for a user with multiple lists with the same title, to potentially find a task associated with a given list title.  This validation makes it so that problem can never occur.